apps/mobile/src/screens/profile/profile.screen.tsx

```diff
@@ -1,19 +1,43 @@
-import { Button } from 'native-base';
+import { Ionicons } from '@expo/vector-icons';
+import { useNavigation } from '@react-navigation/native';
+import { Button, FlatList, Icon, Pressable, Text } from 'native-base';
 
 import Delimiter from '@/components/delimiter/delimiter';
 import SafeArea from '@/components/safe-area/safe-area';
 import Topbar from '@/components/topbar/topbar';
 import { useAuth } from '@/hooks/use-auth-provider';
 import useLocale from '@/hooks/use-locale';
+import Routes from '@/routes';
 
 const ProfileScreen = () => {
   const { signOut } = useAuth();
   const { t } = useLocale();
 
+  const { navigate } = useNavigation();
+
   return (
     <SafeArea>
       <Delimiter flex="1">
         <Topbar />
+        <FlatList
+          data={ROUTES}
+          keyExtractor={({ name }) => `app-route-${name}`}
+          renderItem={({ item }) => (
+            <Pressable
+              onPress={() => navigate(item.path as never)}
+              flexDirection="row"
+              alignItems="center"
+              mt={3}
+              py={4}
+              px={2}
```

Comment:
```suggestion
              mb={2}
```

you should be trying to fix the card alignment here. actually the one that seems misaligned is the arrow in the `TopBar` component, so don't worry with it.